### ISSUE TITLE
cilium-agent: Liveness probe via DaemonSet

### ISSUE THREAD

**Issue by tgraf** [cilium/cilium#893]
Support liveness probe for k8s:

```
          livenessProbe:
            initialDelaySeconds: 30
            httpGet:
              host: 127.0.0.1
              path: /healthz
              port: 8080
```

**Comment by jpds** [comment]
I've created https://github.com/cilium/cilium/pull/938 which fixes this another way.

**Comment by tgraf** [comment]
Fixed by #938
